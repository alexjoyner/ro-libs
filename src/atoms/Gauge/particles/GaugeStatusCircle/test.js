import React from 'react';
import { shallow } from 'enzyme';
import { GaugeStatusCircle } from './';

describe('GaugeStatusCircle component', () => {
  let component;
  beforeEach(() => {
    component = shallow(<GaugeStatusCircle />);
  });
  it('Should render without exploding', () => {
    expect(component).toBeDefined();
  });
  it('Should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
