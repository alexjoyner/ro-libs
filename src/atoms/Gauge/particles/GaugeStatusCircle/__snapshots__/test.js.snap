// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GaugeStatusCircle component Should match snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "GaugeStatusCircle-sc-1qif7k3-0",
        "isStatic": false,
        "rules": Array [
          "stroke:",
          [Function],
          ";stroke-width:17;stroke-dasharray:340.865;stroke-dashoffset:340.865;",
        ],
      },
      "displayName": "GaugeStatusCircle",
      "render": [Function],
      "styledComponentId": "GaugeStatusCircle-sc-1qif7k3-0",
      "target": "circle",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
